Frontend/rasandbox/src/App.vue

```diff
@@ -19,11 +19,16 @@
       </span>
       <span>Filter</span>
     </button>
-    <div class="columns is-gapless is-multiline">
+    <i
+      v-if="isGamesLoading"
+      class="fa-solid fa-spinner fa-4x"
+      :class="{ 'loading-spinner': isGamesLoading }"
```

Comment:
doppelt gemoppeltes class-binding, weil nur bei v-if wird dein <i> tag gerendert, kannst es also in den statischen class="..." string mit aufnehmen